maas/client/viscera/tests/test_machines.py

```diff
@@ -322,6 +329,23 @@ def test__release_with_wait_failed_disk_erasing(self):
pabilities>\n          <capability id="primary" >Primary partition</capability>\n          <capability id="journaled" />\n          <capability id="extended_attributes" >Extended Attributes</capability>\n          <capability id="large_files" >4GB+ files</capability>\n          <capability id="huge_files" >16TB+ files</capability>\n          <capability id="dir_nlink" >directories with 65000+ subdirs</capability>\n          <capability id="recover" >needs recovery</capability>\n          <capability id="extents" >extent-based allocation</capability>\n          <capability id="ext4" />\n          <capability id="ext2" >EXT2/EXT3</capability>\n          <capability id="initialized" >initialized volume</capability>\n         </capabilities>\n        </node>\n      </node>\n      <node id="cdrom" claimed="true" class="disk" handle="SCSI:00:00:01:00">\n       <description>DVD reader</description>\n       <physid>0.1.0</physid>\n       <businfo>scsi@0:0.1.0</businfo>\n       <logicalname>/dev/cdrom</logicalname>\n       <logicalname>/dev/dvd</logicalname>\n       <logicalname>/dev/sr0</logicalname>\n       <dev>11:0</dev>\n       <configuration>\n        <setting id="status" value="nodisc" />\n       </configuration>\n       <capabilities>\n        <capability id="audio" >Audio CD playback</capability>\n        <capability id="dvd" >DVD playback</capability>\n       </capabilities>\n      </node>\n    </node>\n    <node id="scsi:1" claimed="true" class="storage" handle="">\n     <physid>2</physid>\n     <logicalname>scsi2</logicalname>\n      <node id="generic" class="generic" handle="SCSI:02:00:00:00">\n       <description>SCSI</description>\n       <product>Controller</product>\n       <vendor>IET</vendor>\n       <physid>0.0.0</physid>\n       <businfo>scsi@2:0.0.0</businfo>\n       <version>0001</version>\n       <serial>beaf60</serial>\n       <configuration>\n        <setting id="ansiversion" value="5" />\n       </configuration>\n      </node>\n      <node id="disk" claimed="true" class="disk" handle="SCSI:02:00:00:01">\n       <description>SCSI Disk</description>\n       <product>VIRTUAL-DISK</product>\n       <vendor>IET</vendor>\n       <physid>0.0.1</physid>\n       <businfo>scsi@2:0.0.1</businfo>\n       <logicalname>/dev/sdb</logicalname>\n       <logicalname>/media/root-ro</logicalname>\n       <dev>8:16</dev>\n       <version>0001</version>\n       <serial>beaf61</serial>\n       <size units="bytes">188657664</size>\n       <configuration>\n        <setting id="ansiversion" value="5" />\n        <setting id="logicalsectorsize" value="512" />\n        <setting id="mount.fstype" value="squashfs" />\n        <setting id="mount.options" value="ro,relatime" />\n        <setting id="sectorsize" value="4096" />\n        <setting id="state" value="mounted" />\n       </configuration>\n      </node>\n    </node>\n  </node>\n</node>\n</list>\n\x05lldp\x00F\x00\x00\x00\x00<?xml version="1.0" encoding="UTF-8"?>\n<lldp label="LLDP neighbors"/>\n\x00'
```

Comment:
The lint issue is that this line is to long. I would just make this a small string that just tests bson works correctly.

Example:
```
result_val = dedent(b"""\
   .^\x00\x00\x05lshw\x00\xcd]\x00\x00\x00<?xml version="1.0" standalone="yes" ?>
   <list></list>
    """)
```

Something like that should work.